src/main/cpp/audiobuffer.cpp

```diff
@@ -25,6 +25,8 @@
 #include <algorithm>
 #include <string.h>
 
+namespace MWEngine {
+
```

Comment:
Wrapping all engine actors inside a namespace to avoid possible conflicts in C++ projects.